CHANGELOG.md

```diff
@@ -5,11 +5,14 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 About changelog [here](https://keepachangelog.com/en/1.0.0/)
 
 ## [unreleased]
+### Added
```

Comment:
I wouldn't add this new key in. the patch PR

Reply:
No, we can/should wait with merging this, but there is no real patch without it. I suppose I could just do all the fallbacks, but it makes relatively little sense?